### PR TITLE
AARA-398: Implement RFA Tracking POST API

### PR DESCRIPTION
**Pull request by DmitriyRudenko**
### JIRA Issue Link
- https://osi-cwds.atlassian.net/browse/AARA-398

### Tests
- [x] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/AssessmentsDTO.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AssessmentsDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/AssessmentsDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/ClearancesDTO.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ClearancesDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/ClearancesDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/FamilyDocumentsDTO.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FamilyDocumentsDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/FamilyDocumentsDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/IndividualDocumentsDTO.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class IndividualDocumentsDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/IndividualDocumentsDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/TasksAndTrainingsDTO.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TasksAndTrainingsDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/TasksAndTrainingsDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

File: src/main/java/gov/ca/cwds/cals/service/dto/tracking/TrainingsDTO.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
+package gov.ca.cwds.cals.service.dto.tracking;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.io.Serializable;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TrainingsDTO implements Serializable {
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cals/service/dto/tracking/TrainingsDTO.java]: Consider using gov.ca.cwds.cals.service.dto.rfa.collection.CollectionDTO as a superclass

**Comment by DmitriyRudenko** [comment]
done

**Pull request merged**